src/Tribe/JSON_LD/Type.php

```diff
@@ -113,7 +113,10 @@ public function get_data( $posts = null, $args = array() ) {
 				continue;
 			}
 
-			$data = parent::get_data( $post, $args );
+			$data = array();
+			if ( function_exists( 'tribe_events_get_the_excerpt' ) ) {
```

Comment:
I mean we should add the fix on Common to avoid calling `tribe_events_get_the_excerpt` if not available, fall back on the WordPress `$post->post_excerpt`.

Reply:
Closing this PR here then an opening one on TEC as is where the parent class is located.